### ISSUE TITLE
Remove/Rework ISIS id in the datafiles table in upc_model.py

### ISSUE THREAD

**Issue by acpaquette** [USGS-Astrogeology/PDS-Pipelines#395]
Since products will be coming in from other data sources it may be worth while to think of removing/retooling the ISISid in the upc databse.

Questions to answer (Definition of done when answered):
- Does UPC allow the ISIS ID to be null?
- Can we upc process the file successfully without an ISIS ID?

**Comment by kaitlyndlee** [comment]
It looks like upc_process sets the isis id to null here: https://github.com/USGS-Astrogeology/PDS-Pipelines/blob/777f7a5bc96bbb21f5a4df0bd6bcb65baef80331/pds_pipelines/upc_process.py#L266. Looking at the upc tests when testing when no isis id exists, getsn should throw an Attribute Error. If getsn throws a ProcessError and KeyError, then we set the sn to an empty string here: https://github.com/USGS-Astrogeology/PDS-Pipelines/blob/777f7a5bc96bbb21f5a4df0bd6bcb65baef80331/pds_pipelines/upc_process.py#L73. 

Does anyone know why we set the sn to an empty string when those errors are thrown? If getsn can't find a sn, wouldn't it throw a ProcoessError since the isis command would fail when trying to find it or am I thinking about this wrong?

The only other places isis id is used is in the browse_process and thumbnail_process scripts. In these scripts, they call getsn, but do not catch any exceptions or set the isis id to null.
